src/ShapePopulationData.cxx

```diff
@@ -1,5 +1,41 @@
 #include "ShapePopulationData.h"
 
+#include <qSlicerCoreApplication.h>
+#include <vtkMRMLScene.h>
+#include <vtkMRMLSRepNode.h>
+#include <vtkMRMLSRepStorageNode.h>
+#include <vtkSlicerApplicationLogic.h>
+#include <vtkSlicerSRepLogic.h>
+#include <vtkSRepExportPolyDataProperties.h>
```

Comment:
@Connor-Bowley I am building the SPV PR branch using the current SlicerSALT master, the build couldn't find vtkSRepExportPolyDataProperties.h, could you check where this file is?

Reply:
I see that this file is in your SRep2 local branch. We probably may want to merge it to the SRep2 master, update SlicerSALT master, and check from there.

Reply:
Sounds good.